_i18n/en.yml

```diff
@@ -326,13 +338,28 @@ help_pages:
+
+      Recently we have began the process of retiring personal keys as a form of two factor authentication as they are apt to being lost easily. In their place users now have a variety of options for a second authentication factor including backup codes.
+
+      If you are creating a new account you will not be assigned a personal key. Instead, select one of the available two-factor authentication options presented.
 
-      You can only use your personal key once.  You will receive a new personal key each time you use it to sign in, and your old one will no longer work.
+      If you were assigned a personal key previously you are still able to use it, however next time you use the personal key you will be asked to select a new form of two factor authentication and the personal key option will be removed. Until you use your current personal key it will still be active and available to you as a two factor option.
 
-      Because a personal key is the only way to recover your account, make sure to store it in a safe place. Some good ways of storing your personal key include writing it down, taking a screenshot, pasting it into a secure document, or making a printout of it.
+      <!---
+      TO ADD WHEN DEPLOYING DOCAUTH
+      You are assigned a personal key when you complete your identification proofing. You can only use your personal key once.  You will receive a new personal key each time you use it to sign in, and your old one will no longer work.
+
+      Because a personal key is the only way to reset your password while maintaining your identity validated, make sure to store it in a safe place. Some good ways of storing your personal key include writing it down, taking a screenshot, pasting it into a secure document, or making a printout of it. If you do forget your personal key you will still be able to recover your account but must go through the proofing process again.
 
       [Read more on how to sign in using your personal key](site.baseurl/help/signing-in/if-i-dont-have-my-phone-with-me-can-i-still-sign-in)
+      -->
+    what-are-backup-codes: Backup codes are a set of 10 generated codes you can use as a second authentication mechanism. If you lose your phones or otherwise can't get codes via text, voice call, or authentication app, you can use backup codes to sign in. If you don't have your security key, you can also use these codes to sign in.
+
+      After you’ve used a backup code to sign in, it will become inactive. Once you have used all of your backup codes a new set will be generated for you. You can also generate a new set of 10 codes from the account page. After creating a new set, the old set will automatically become inactive.
+
+      We recommend you store your codes in a safe manner. Like the codes on your phone, backup codes are only valuable to someone if they manage to also steal your password.
```

Comment:
> Like the codes on your phone, backup codes are only valuable to someone if they manage to also steal your password.

Suggested: Like the security codes sent to your phone, backup codes are only valuable to someone if they manage to also steal your password.

Reply:
Changed